packages/notifications-flyout/src/behaviors/NotificationFlyoutBehavior.test.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { mount } from "enzyme";
+
+import NotificationFlyoutBehavior from "./NotificationFlyoutBehavior";
+import Notification from "../Notification";
+import NotificationsFacade from "../facades/NotificationsFacade";
+
+describe("notification-flyout/behaviors/NotificationFlyoutBehavior", () => {
+  const children = jest.fn();
+
+  function mountBehavior() {
+    return mount(
+      <NotificationFlyoutBehavior
+        notifications={
+          <NotificationsFacade>
```

Comment:
I'm unclear on what `NotificationsFacade` is about. It has a `children` but it returns null. What is it doing?

Reply:
The `NotificationsFacade` component was to allow for consumers to use the `notifications` prop without the use of an array. However, now looking at it, it's redundant when the `children` prop is also available. I've removed it as this can be considered over-engineering and can be added later if need be.